server/src/main/java/org/elasticsearch/common/logging/ESJsonLayout.java

```diff
@@ -33,21 +33,29 @@
 import java.nio.charset.Charset;
 import java.util.Map;
 
+/**
+ * Formats log events as strings in a json format.
+ * <p>
+ * The class is wrapping the {@link PatternLayout} with a pattern to format into json. This gives more flexibility and control over how the
+ * log messages are formatted in {@link org.apache.logging.log4j.core.layout.JsonLayout}
+ */
 @Plugin(name = "ESJsonLayout", category = Node.CATEGORY, elementType = Layout.ELEMENT_TYPE, printObject = true)
 public class ESJsonLayout extends AbstractStringLayout {
-
     /**
-     * type - the type of logs. These represent appenders and help docker distinguish log streams.
-     * timestamp - ISO8601 with additional timezone ID
-     * level - INFO, WARN etc
-     * component - logger name, most of the times class name
-     * cluster.name - taken from sys:es.logs.cluster_name system property because it is always set
-     * node.name - taken from NodeNamePatternConverter, as it can be set in runtime as hostname when not set in elasticsearch.yml
-     * node_and_cluster_id - in json as node.id and cluster.uuid - taken from NodeAndClusterIdConverter and present
+     * Fields used in a pattern to format a json log line:
+     * <ul>
+     * <li>  type - the type of logs. These represent appenders and help docker distinguish log streams.
```

Comment:
is this actually valid? I think the closing tags are missing, i.e. the structure needs to be `<li>Text</li>` instead of `<li>Text`. Also nit: Remove leading space for each item.

Reply:
It does work indeed, but I guess better to stick with the html format. Also the space removed.